vector/src/main/java/im/vector/app/core/pushers/VectorPushHandler.kt

```diff
@@ -45,28 +42,20 @@ import org.matrix.android.sdk.api.logger.LoggerTag
 import org.matrix.android.sdk.api.session.Session
 import org.matrix.android.sdk.api.session.getRoom
 import org.matrix.android.sdk.api.session.room.getTimelineEvent
-import org.unifiedpush.android.connector.MessagingReceiver
 import timber.log.Timber
 import javax.inject.Inject
 
 private val loggerTag = LoggerTag("Push", LoggerTag.SYNC)
 
-/**
- * Hilt injection happen at super.onReceive().
- */
-@AndroidEntryPoint
-class VectorMessagingReceiver : MessagingReceiver() {
-    @Inject lateinit var notificationDrawerManager: NotificationDrawerManager
-    @Inject lateinit var notifiableEventResolver: NotifiableEventResolver
-    @Inject lateinit var pushersManager: PushersManager
-    @Inject lateinit var activeSessionHolder: ActiveSessionHolder
-    @Inject lateinit var vectorPreferences: VectorPreferences
-    @Inject lateinit var vectorDataStore: VectorDataStore
-    @Inject lateinit var wifiDetector: WifiDetector
-    @Inject lateinit var guardServiceStarter: GuardServiceStarter
-    @Inject lateinit var unifiedPushHelper: UnifiedPushHelper
-    @Inject lateinit var unifiedPushStore: UnifiedPushStore
-    @Inject lateinit var pushParser: PushParser
+class VectorPushHandler @Inject constructor(
```

Comment:
@Florian14 can you please have a look on this constructor? 
the resulting constructor is different than the Element-Android [here](https://github.com/vector-im/element-android/pull/7085/files#diff-7cfab86ce963c810b26123032e087558d4f4c271d29a67576494933465276de2R50)

BTW The existing code `VectorMessagingReceiver` was already different. So this may be normal. I'm curious to know the reason of these existing differences.

Reply:
`actionsIds` has been added in [this commit](https://github.com/vector-im/element-android/commit/811b2e8c75746878b651ee046ee31f1312eed745#diff-89945d3e11caf4a6cddd30ff052082001992c86186660ee2e1c9093f04c87f30R72) in EA-v1.4.32
`buildMeta` has been added in [this commit](https://github.com/vector-im/element-android/commit/d42c27bfe0a13699e9b8ea6de107dff1c322ff1c#diff-89945d3e11caf4a6cddd30ff052082001992c86186660ee2e1c9093f04c87f30) also in EA-v1.4.32

Current Tchap-Android release is rebased on EA-v1.4.27-RC2